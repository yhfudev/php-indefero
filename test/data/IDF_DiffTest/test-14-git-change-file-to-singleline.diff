diff --git a/b b/b
index bfdaa0f..7898192 100644
--- a/b
+++ b/b
@@ -1,3 +1 @@
 a
-b
-
