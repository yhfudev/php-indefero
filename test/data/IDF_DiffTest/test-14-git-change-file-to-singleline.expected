<?php return array (
  'b' => 
  array (
    'chunks' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '1',
          1 => '1',
          2 => "a\n",
        ),
        1 => 
        array (
          0 => 2,
          1 => '',
          2 => "b\n",
        ),
        2 => 
        array (
          0 => 3,
          1 => '',
          2 => "\n",
        ),
      ),
    ),
    'chunks_def' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '1',
          1 => '3',
        ),
        1 => 
        array (
          0 => '1',
          1 => 1,
        ),
      ),
    ),
  ),
);